octodns/provider/cloudflare.py

```diff
@@ -66,16 +64,16 @@ class CloudflareProvider(BaseProvider):
     MIN_TTL = 120
     TIMEOUT = 15
 
-    def __init__(self, id, email, token, cdn=False, *args, **kwargs):
+    def __init__(self, id, token, cdn=False, *args, **kwargs):
```

Comment:
I think we'll want to go about this in a way that doesn't break existing usage of email & token. That'd likely involve making `email` an optional parameter that defaults to `None` and then if'ing some handling details down below.

Reply:
Yeah, I actually had the same thought the other day... I'll fix it up to do both.